project/model.py

```diff
@@ -0,0 +1,78 @@
+from torch.nn import Linear, Tanh, LSTM, Embedding
+from torch.optim import SGD
+import pytorch_lightning as pl 
+from torchvision.models import resnet18
+from loss_func.simclr import SimCLR
+import torch
+
+class JeopardyModel(pl.LightningModule):
+    def __init__(self, vocab_sz, bs):
+      super().__init__()
+      im_vec_dim = 128
+      ans_dim = 128
+      question_dim = 256
+      n_hidden = 100
+      n_layers = 1
+      
+      self.i_h = Embedding(vocab_sz, n_hidden)
+      self.rnn = LSTM(n_hidden, n_hidden)
+      self.h_o = Linear(n_hidden, question_dim)
+
+      self.h = None 
+      self.ans_final = Linear(n_hidden, ans_dim)
+
+      # for images, we use resnet18 pretrained
+      self.image_feature_extractor = resnet18(pretrained=True, progress=True)
```

Comment:
pretrained should not be set to true, we are learning frmo scratch. don't set progress to true either